doc/source/user_guide/ug_case_settings.rst

```diff
@@ -49,14 +54,19 @@ to support the CICE model.
    "ICE_BASECOM", " ", "baseline directory name for regression comparison, associated with cice.setup -bcmp ", "set by cice.setup"
    "ICE_BFBCOMP", " ", "location of case for comparison, associated with cice.setup -td", "set by cice.setup"
    "ICE_SPVAL", " ", "special value for cice.settings strings", "set by cice.setup"
-   "ICE_RUNLENGTH", " ", "batch run length default", "set by cice.setup"
+   "ICE_RUNLENGTH", "integer", "batch run length default", "set by cice.setup"
+   " ", "0", "15 minutes (default)", " "
+   " ", "1", "59 minutes", " "
+   " ", "2", "2 hours", " "
+   " ", "other :math: 2 < N < 8", "N hours", " "
+   " ", "8 or larger", "8 hours", " "
    "ICE_ACCOUNT", " ", "batch account number", "set by cice.setup, .cice_proj or by default"
    "ICE_QUEUE", " ", "batch queue name", "set by cice.setup or by default"
-   "ICE_THREADED", "true,false", "force threading in compile, will always compile threaded if NTHRDS is gt 1", "false"
+   "ICE_THREADED", "true, false", "force threading in compile, will always compile threaded if ICE_NTHRDS :math: > 1", "false"
```

Comment:
You'll have to add ` signs around the >1 (this didn't render before in my comment, sorry!)